src/content/en/2021/jamstack.md

```diff
@@ -1,20 +1,506 @@
+) }}
+
+### Summary of adoption by regions
+
+SSG technology is adopted not only by places where it’s invented and popularized, but also all around the world. Moreover, in places where the next generation of the web will be built, it grows even faster, mostly 2 times and sometimes 4 times, compared to others.
+
+## SSGs distribution among CDN providers
+
+We described how SSGs are adopted in different countries, so let's analyse which SSG is most popular among different CDN providers.
+
+There’s 7 most popular CDN providers for SSGs
+
+* Netlify
+* Vercel
+* Cloudflare
+* AWS
+* Azure
+* Akamai
+* GitHub
+
+Note: GitHub as CDN provider might be questioned, but GitHub offers a feature called [GitHub Pages](https://pages.github.com/) which allows users to deploy sites on github.io with subdomain using built in Jekyll SSG.
+
+{{ figure_markup(
+  image="ssg-distribution-over-cdn.png",
+  caption="SSG distribution over CDN",
+  description="Bar chart represents SSG distribution over CDN.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRUiAdcUCSYyS7cQcQlE9uKeMSacgRudtf4pFRkIKKCt8Nw18qOod6TaAUL7tfm8pUbCPKShEN0jJOS/pubchart?oid=1366870673&format=interactive",
+  sheets_gid="1565107748",
+  sql_file="core_web_vitals_distribution.sql"
+) }}
+
+Next.js, as most popular by data we have, mostly served by Cloudflare, Vercel, and AWS. Most of Gatsby's sites use Netlify, AWS, and Cloudflare.
+
+Nuxt.js sites preferred to be served by Cloudflare, AWS, and Netlify.
+
+Hugo mostly uses Netlify, and Jekyll as no surprise mostly on GitHub.
+
+Fair to say that in real world, Cloudflare is used not only for pure serving as CDN but as protection service, DNS balancing etc. So, keep in mind that Cloudflare results can be false positives.
+
+Putting Cloudflare aside for a while, we can see interesting data about Next.js. Next.js is mostly served by Vercel (company invented Next.js), and also Next.js heavily served by AWS. Together with Nuxt.js and Gatsby, we can define that AWS as a Jamstack oriented service has made great progress on serving Jamstack sites together with Jamstack oriented services like Netlify and Vercel.
+
+On following graph it’s more clear which SSGs mostly delivered by which CDN.
+
+{{ figure_markup(
+  image="ssg-distribution-over-cdn-stacked.png",
+  caption="SSG distribution over CDN",
+  description="Stacked Bar chart represents SSG distribution over CDN.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRUiAdcUCSYyS7cQcQlE9uKeMSacgRudtf4pFRkIKKCt8Nw18qOod6TaAUL7tfm8pUbCPKShEN0jJOS/pubchart?oid=180251589&format=interactive",
+  sheets_gid="1565107748",
+  sql_file="core_web_vitals_distribution.sql"
+) }}
+
+## Performance
+### Lighthouse
+
+All Lighthouse scores are median values for mobile devices. Hence, these are results from all around the world. There might be influence of using mobile data providers or devices with simpler hardware then top edge devices of 2021.
+#### Performance
```

Comment:
Two headings both called Performance. Avoid this.

(also stacked heading as mentioned before)